src/devtools/client/debugger/src/actions/breakpoints/breakpoints.ts

```diff
@@ -59,10 +59,12 @@ export function removeBreakpointsAtLine(
   return (dispatch, getState) => {
     trackEvent("breakpoint.remove");
 
-    dispatch(removeRequestedBreakpoint({ sourceId, line }));
     const breakpoints = getBreakpointsForSource(getState(), sourceId, line);
 
-    breakpoints.map(bp => dispatch(removeBreakpoint(cx, bp)));
+    breakpoints.map(bp => {
+      dispatch(removeBreakpoint(cx, bp));
```

Comment:
:hand: Any chance we could do this as _one_ action, `removeBreakpoint`, and have the "requested" handle that part too?  Not a big deal, and doesn't have to be done in this PR.

Reply:
Good question. I think we probably could.